frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/groupActionMenu/groupSettingsForm/groupSettingsForm.component.tsx

```diff
@@ -0,0 +1,205 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { formatMessage } from '@/v5/services/intl';
+import { ProjectsHooksSelectors } from '@/v5/services/selectorsHooks';
+import { FormColorPicker, FormTextField } from '@controls/inputs/formInputs.component';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { FormattedMessage } from 'react-intl';
+import { SubmitButton } from '@controls/submitButton';
+import { Button } from '@controls/button';
+import { IGroupSettingsForm } from '@/v5/store/tickets/groups/ticketGroups.types';
+import { useContext, useState } from 'react';
+import { GroupSettingsSchema } from '@/v5/validation/groupSchemes/groupSchemes';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { isEmpty } from 'lodash';
+import { ActionMenuItem } from '@controls/actionMenu';
+import { Buttons, LabelAndColor, FormBox, Heading, Instruction, CreateCollectionLink, Subheading, ToggleLabel, ToggleWrapper, Toggle, FormRow } from './groupSettingsForm.styles';
+import { TicketGroupsContext } from '../../../ticketGroupsContext';
+
+export const GroupSettingsForm = ({ defaultValues }: { defaultValues?: IGroupSettingsForm }) => {
+	const [isSmart, setIsSmart] = useState(!!defaultValues?.rules?.length);
+	const isHidden = useContext(TicketGroupsContext).groupType === 'hidden';
+	const {
+		control,
+		handleSubmit,
+		formState: { errors, isValid, dirtyFields },
+	} = useForm<IGroupSettingsForm>({
+		mode: 'onChange',
+		resolver: yupResolver(GroupSettingsSchema),
+		defaultValues,
+	});
+	const isAdmin = ProjectsHooksSelectors.selectIsProjectAdmin();
+	const isNewGroup = !defaultValues?._id;
+
+	const onSubmit: SubmitHandler<IGroupSettingsForm> = (args) => {
+		// eslint-disable-next-line no-console
+		console.log({ args, isHidden });
+	};
+	return (
+		<form>
```

Comment:
The width of the form in the mockup is 394: 
<img width="886" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/be8bb2ba-8152-436c-b8d5-7a447210e408">


While here is closer to 360

<img width="361" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/426ae94c-eb94-4abc-9972-09ca51954cd9">